tests/db/migration_tests/test_v202411140_to_v202501140.py

```diff
@@ -0,0 +1,37 @@
+"""
+    Copyright 2024 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+
+import os
+import re
+from collections import abc
+
+import asyncpg
+import pytest
+
+file_name_regex = re.compile("test_v([0-9]{9})_to_v[0-9]{9}")
+part = file_name_regex.match(__name__)[1]
+
+
+@pytest.mark.parametrize("no_agent", [True])
+@pytest.mark.db_restore_dump(os.path.join(os.path.dirname(__file__), f"dumps/v{part}.sql"))
+async def test_add_new_resource_status_column(
+    postgresql_client: asyncpg.Connection,
+    migrate_db_from: abc.Callable[[], abc.Awaitable[None]],
+) -> None:
+    # only test that the migration script succeeds. Actual migration is covered by migration test for v202411140
+    await migrate_db_from()
```

Comment:
Given this comment:
1. does the other test cover this sufficient? I would assume it does?  i.e. does the other test fail if you remove the upgrade code?
1. why run this at all, it takes time and adds no value? It doesn't even check if the scheduler could start. Make it an empty file with this comment in it?

Reply:
1. As far as I can tell, yes.
2. Fair enough, we need the file but not necessarily the migration.

Reply:
Fixed in 306f69e